_config.yml

```diff
@@ -29,6 +29,9 @@ navigation:
 - text: Collaborating on Code
```

Comment:
this is more of a future thought....but it seems like this other page's name should change. Code review could also be considered a form of collaboration.